_episodes/02-the-filesystem.md

```diff
@@ -1,32 +1,22 @@
 ---
-title: "The Filesystem"
-teaching: 0
-exercises: 0
+title: "Navigating Files and Directories"
+teaching: 30
+exercises: 20
 questions:
-- How can I give my computer directions to files I want to access?  
-- Ho
+- "How can I perform operations on files outside of my working directory?"
+- "What are some navigational shortcuts I can use to make my work more efficient?"
 objectives:
-- Navigate the Unix file system
-- Understand and use file paths and relative paths
-- Keyboard shortcuts to make work more efficiently at the command line 
+- "Use a single command to navigate multiple steps in your directory structure, including moving backwards (up a level)."
```

Comment:
I would prefer: Use a single command to navigate multiple steps in your directory structure, including moving backwards (one level up)